charts/backbeat-api/templates/deployment.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "backbeat-api.fullname" . }}
+  labels:
+    app: {{ template "backbeat-api.name" . }}
+    chart: {{ template "backbeat-api.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "backbeat-api.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: backbeat-api
+              containerPort: 8900
+          env:
+            - name: ZOOKEEPER_AUTO_CREATE_NAMESPACE
+              value: "1"
+            - name: ZOOKEEPER_CONNECTION_STRING
+              value: "{{- printf "%s-zenko-zookeeper:2181" .Release.Name | trunc 63 | trimSuffix "-" -}}"
+            - name: KAFKA_HOSTS
+              value: "{{- printf "%s-kafka-0.%s-kafka:9092" .Release.Name .Release.Name | trunc 63 | trimSuffix "-" -}}"
+            - name: REDIS_HOST
+              value: "{{- printf "%s-%s" .Release.Name "redis" | trunc 63 | trimSuffix "-" -}}"
+            - name: REDIS_PORT
+              value: "6379"
+            - name: HEALTHCHECKS_ALLOWFROM
+              value: '0.0.0.0/0'
```

Comment:
Probably this should be limited eventually to relevant monitoring hosts only, but ok for now.

Reply:
If we ever want to limit inter-service access, we'd better specify [NetworkPolicies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) rather than provide partial implementations of some kind of firewall in every single service.